app/models/manageiq/providers/openstack/network_manager/network_router.rb

```diff
@@ -2,10 +2,26 @@ class ManageIQ::Providers::Openstack::NetworkManager::NetworkRouter < ::NetworkR
   include ProviderObjectMixin
   include AsyncDeleteMixin
 
-  supports :create_network_router
-  supports :delete_network_router
-  supports :update_network_router
```

Comment:
@durandom, oh, unless you mean moving to the base class :). I'll look into that one.